internal/api/arm/subscription.go

```diff
@@ -48,7 +48,7 @@ type SubscriptionProperties struct {
 	SpendingLimit        *string              `json:"spendingLimit,omitempty"`
 	AccountOwner         *AccountOwner        `json:"accountOwner,omitempty"`
 	ManagedByTenants     *[]map[string]string `json:"managedByTenants,omitempty"`
-	AdditionalProperties *map[string]string   `json:"additionalProperties,omitempty"`
+	AdditionalProperties any                  `json:"additionalProperties,omitempty"`
```

Comment:
If we ever want to read anything from `AdditionalProperties`, defining it as `map[string]any` would be better. But for now it doesn't matter.